### PR TITLE
NPSP Soft Credit Recipe Streamline

### LINKED ISSUE #89: Bug: Oppty NPSP Soft Credit Recipe does not create OCRs

### PR DESCRIPTION
**Pull request by allisonletts**
# Critical Changes
- restructures for clearer relationships

# Changes

# Issues Closed
fixes #89

### CODE REVIEW

File: snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml [hunk 1]
```diff
@@ -2,44 +2,30 @@
...
 
 ### ----------- [               CLI RUN                ]---------- ###
-# Use this command to run this recipe 300 times in a Salesforce scratch org called "dev":
-# cci task run generate_and_load_from_yaml -o generator_yaml snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml -o num_records 300 -o num_records_tablename Account --org dev
-#
-# Use this command to run this recipe locally and output to a JSON file called output.json
-# snowfakery --output-format json --output-file snowfakery_samples/temp/output.json snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml
-#
+
+# cci task run snowfakery --recipe snowfakery_samples/npsp/Opportunity_npsp_soft_credits.yml
```
Comment (jesslopez-sf) [inline snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml:13]:
```suggestion
# cci task run snowfakery --recipe snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml
```

File: snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml [hunk 2]
```diff
@@ -2,44 +2,30 @@
...
+# snowfakery --output-format json --output-file src/foo.json snowfakery_samples/npsp/Opportunity_npsp_soft_credits.yml
```
Comment (jesslopez-sf) [inline snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml:14]:
```suggestion
# snowfakery --output-format json --output-file snowfakery_samples/temp/output.json snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml
```

File: snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml [hunk 3]
```diff
@@ -2,44 +2,30 @@
 
 ### ------------- [          ORDER OF INSERT         ]------------ ###
 # 1-0. Account (1)
-# 1-1. Contact + Opportunity + OpportunityContactRole (5)
+# 1-1. Opportunity + OpportunityContactRole + Contact (5)
 
 ### ----------- [ Opportunity_npsp_soft_credit.recipe.yml Summary ]---------- ###
-# Create a set of 5 Opportunities with an Opportunity Contact Role and Contact under 1  Account.
+# Create a set of 5 Opportunities with an Opportunity Contact Role and Contact under 1 Account.
+# The account
```
Comment (jesslopez-sf) [inline snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml:9]: Not sure if this was a comment left over from the original recipe or if part of it was accidentally deleted?

**Review by jesslopez-sf — CHANGES_REQUESTED** [review]
@allisonletts - the recipe worked like a charm! I added some super minor suggestions, mainly around the commands you would use to run the recipe. The name of the recipe file didn't match and the directory to output the temp json file was also changed. Otherwise, great job! :)

File: snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml (hunk 3 above)
Comment (allisonletts) [reply snowfakery_samples/npsp/Opportunity_npsp_soft_credit.recipe.yml]: whoops! Thanks!

**Review by jesslopez-sf — APPROVED** [review]
Awesome. Thank you! :)

**Pull request merged**